client/package.json

```diff
@@ -58,6 +58,6 @@
   ],
   "engines": {
     "npm": ">=8.0.0",
-    "node": ">=17.0.0"
+    "node": ">=16.0.0"
```

Comment:
Curious why this change?

Reply:
Node 17 isn't supported anymore and a dependency requires Node <=17. So 16 is the most recent, supported version.

(I had to make the same change in my [other PR](https://github.com/BlueConduit/open-data-platform/pull/55) but forgot to call it out here)